examples/lines_bars_and_markers/joinstyle.py

```diff
@@ -14,10 +14,10 @@ def plot_angle(ax, x, y, angle, style):
     phi = np.radians(angle)
     xx = [x + .5, x, x + .5*np.cos(phi)]
     yy = [y, y, y + .5*np.sin(phi)]
-    ax.plot(xx, yy, lw=8, color='blue', solid_joinstyle=style)
+    ax.plot(xx, yy, lw=8, color='C0', solid_joinstyle=style)
```

Comment:
This has nothing to do with the colors in the cycle. We're just selecting two reasonable colors. Therefore, I'd rather go with 'tab:blue' and 'tab:red' instead of 'C0', 'C3' in this case.

Reply:
Done that where possible; it's doesn't work for format strings (ie `C0--`), so I can either get rid of those by splitting multiple data-sets to `plot` into multiple calls or leave as is?

Reply:
Not a friend of the plot-multiple-datasets-with-one-command API. It‘s less clear except for the most simple use-cases. It‘s less powerful. And it‘s yet another thing users have to learn.

So, I‘m generally in favor of reducing its use in the examples. However, I don‘t insist on it.